### PR TITLE
insert thought provoking commit message here

### PR DESCRIPTION
**Pull request by jpostenwapa**
Updates my slide per the instructions.

## Done
- file changed
- file saved
- committed
- pushed

## Not Done Yet
- [x] reject @industrylol PR

### CODE REVIEW

**Comment by larrybaugh** [comment]
Worst code ever!!

**Review by larrybaugh — APPROVED** [review]
No time for real QA, let's go to Prod!  Approved!

**Comment by hannalyst** [comment]
incrementally refreshing my approval

**Review by hannalyst — APPROVED** [review]

File: _slides/10-jpostenwapa.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Caption here, caption there, everywhere a caption!
```
Comment (industrylol) [inline _slides/10-jpostenwapa.md:6]: This is too overly broad, isn't this supposed to cover a specific use case?

**Review by industrylol — APPROVED** [review]

**Comment by jpostenwapa** [comment]
fixing #2

**Pull request merged**